src/ctl_context.rs

```diff
@@ -48,24 +53,33 @@ impl CtlContext {
     }
 }
 
+/// Used to build an encoded `CtlContext` which can be added to a `Memory` store
+/// to get back the actual `CtlContext`.
 pub struct Builder {
     certificates: Vec<CertContext>,
     usages: Vec<Vec<u8>>,
 }
 
 impl Builder {
```

Comment:
We'll need to allow signers to be added before it makes sense for this to be public.